ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -630,7 +631,20 @@ def set_upgrade_images(self):
             resource_name=constants.OPERATOR_CATALOG_SOURCE_NAME,
             namespace=constants.MARKETPLACE_NAMESPACE,
         )
-
+        stage_testing = config.DEPLOYMENT.get("stage_rh_osbs")
+        konflux_build = config.DEPLOYMENT.get("konflux_build")
+        if konflux_build and stage_testing:
+            log_step("Creating stage TagMirrorSet")
+            exec_cmd(f"oc apply -f {constants.STAGE_TAG_MIRROR_SET_YAML}")
+            log_step("Creating stage ImageDigestMirrorSet")
+            exec_cmd(f"oc apply -f {constants.STAGE_IMAGE_DIGEST_MIRROR_SET_YAML}")
```

Comment:
is order of applying STAGE_TAG_MIRROR_SET_YAML and STAGE_IMAGE_DIGEST_MIRROR_SET_YAML has any affect. during deployment STAGE_IMAGE_DIGEST_MIRROR_SET_YAML was applied 1st and then STAGE_TAG_MIRROR_SET_YAML

Reply:
It should not matter